packages/yoga/src/Dialog/web/Dialog.jsx

```diff
@@ -85,7 +85,12 @@ const Dialog = ({ isOpen, children, onClose }) => {
 
   return isOpen ? (
     createPortal(
-      <Overlay onClick={closeDialog} onClose={onClose} ref={dialogRef}>
+      <Overlay
+        onClick={closeDialog}
+        onClose={onClose}
+        ref={dialogRef}
+        {...props}
```

Comment:
I don't think we have advantages on being able to customize this overlay, wouldn't be better if we passed this directly to `StyledDialog`?

Reply:
I couldn't do this without creating a prop in the Dialog component or making some drastic change. Can we refine this and discuss it at another time?